pkg/app/kafka.go

```diff
@@ -370,49 +435,133 @@ func K8SServicePortForward(ctx context.Context, svcName string, ns string, pPort
 	return f, nil
 }
 
-func consume(ctx context.Context, topic string, namespace string) (int, error) {
-	partition := 0
-	forwarder, err := K8SServicePortForward(ctx, bootstrapServerHost, namespace, bootstrapServerPort)
+// createConsumerGroup creates a consumer group in kafka cluster
+func createConsumerGroup(uri string) error {
+	data := Payload{
+		Name:                     topic + "-consumer",
```

Comment:
Why are we adding `-consumer` here? We are doing same while constructing URL as well. Can we just use topic and avoid combining string `topic+"-consumer"` while building URLs also?

Reply:
yeah we can do that, but we have appended `-consumer` as it would differentiate the `topic` and `consumer-group` instances that would subscribe to the topic. So its just a format we are using here.